### DISCUSSION TITLE
error case

### DISCUSSION THREAD

**Discussion by byungmeo** [byungmeo/GersangStation#7] [Announcements]
https://byungmeo.github.io/GersangStation/

이 페이지는 이제 사용되지 않습니다. 자주 묻는 질문 탭을 참고 해주세요.

**Comment by D0iloppa** [comment]
마지막 에러 
'액세스 권한에 의해 숨겨진 소켓에 액세스를 시도했습니다.'
발생하는 경우는 왜일가요?

포트충돌인가요?
VS code, VM, hyper-V 말고는 백그라운드 프로세스는 없습니다.

**Reply by D0iloppa** [reply]
해당 오류는 
GersangStation의 에러는 아니고 거상 클라이언트 자체 에러인 것으로 확인되었습니다.
